packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -312,7 +312,7 @@ const GeneralTab = decorate([
         <br />
         <Row className='text-xs-center'>
           <Col>
-            {vm.boot.firmware === 'uefi' && semver.satisfies(host.version, '>=8.3.0') && (
+            {vm.boot.firmware === 'uefi' && semver.satisfies(host?.version, '>=8.3.0') && (
```

Comment:
I think it would make more sense to test explicitly `host !== undefined` otherwise a test like `!semver.satisfies(host?.version, '<8.3.0')` would be `true` for non-admin users, which doesn't really make sense.

Reply:
Wouldn't it make the feature appear on hosts older than 8.3 ?